pkg/common/oras/authprovider/azure/azureworkloadidentity.go

```diff
@@ -139,12 +139,23 @@ func (d *azureWIAuthProvider) Provide(ctx context.Context, artifact string) (pro
 	}
 	metrics.ReportACRExchangeDuration(ctx, time.Since(startTime).Milliseconds(), artifactHostName)
 
+	// set default refresh token expiry to default ACR expiry - 5 minutes
+	acrExpiration := time.Now().Add(defaultACRExpiryDuration - 5*time.Minute)
```

Comment:
did we want to add this similar logic to the azure identity provider? I'm fine either way you decide but thought I'd mention it.